plugins/builtin/apm/datadog/plugin/plugin.go

```diff
@@ -0,0 +1,146 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad-autoscaler/plugins"
+	"github.com/hashicorp/nomad-autoscaler/plugins/apm"
+	"github.com/hashicorp/nomad-autoscaler/plugins/base"
+)
+
+const (
+	// pluginName is the name of the plugin
+	pluginName = "datadog"
+
+	configKeyClientAPIKey = "dd_client_api_key"
+	configKeyClientAPPKey = "dd_client_app_key"
```

Comment:
I am curious here on your thoughts of dropping the `client` section also expanding `dd` to `datadog` to ensure there are no clashes or confusion in the future?

Reply:
Datadog prefers `DD_` prefix in its own API/Code

For consistency with Datadog they should be named like this

```go
configKeyClientAPIKey = "dd_api_key"
configKeyClientAPPKey = "dd_app_key"
```

Also, would be great if these values could be read from env :)

Reply:
Sounds good, I'll add those changes